langchain/src/callbacks/base.ts

```diff
@@ -408,6 +504,12 @@ export class CallbackManager extends BaseCallbackManager {
     this.handlers = handlers;
   }
 
+  getChild(): CallbackManager {
+    const manager = new CallbackManager(this._currentRunId);
```

Comment:
We can actually remove need for any state tracking currentRunId here if we take it as an arg to getChild

Reply:
The caller of getChild will always have its own runId around somewhere

Reply:
I think it's better for getChild to not have any arguments, no?